pkg/private/manifest.py

```diff
@@ -49,9 +49,16 @@ def __repr__(self):
 
 def read_entries_from(fh):
     """Return a list of ManifestEntry's from `fh`"""
-    raw_entries = json.load(fh)
+    # Subtle: decode the content with read() rather than in json.load() because
+    # the load in older python releases (< 3.7?) does not know how to decode.
+    raw_entries = json.loads(fh.read())
```

Comment:
Hah, fun fun.

Reply:
This took a lot of head scratching to discover.